content/pt-br/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Anexando Manipuladores aos Eventos de Ciclo de Vida do Contêiner
+content_type: task
+weight: 140
+reviewers:
+---
+
+<!-- overview -->
+
+Esta página mostra como anexar manipuladores aos eventos de ciclo de vida do contêiner. 
+O Kubernetes suporta os eventos `postStart` e `preStop`. O Kubernetes envia o evento 
+`postStart` imediatamente após um contêiner ser iniciado, 
+e envia o evento `preStop` imediatamente antes do contêiner ser terminado. 
+Um contêiner deve especificar um manipulador por evento.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+<!-- steps -->
+
+## Definindo manipuladores para `postStart` e `preStop`
+
+Neste exercício, você cria um Pod que tem um contêiner. 
+O contêiner tem um manipulador para os eventos `postStart` e `preStop`.
+
+Aqui está o arquivo de configuração para o Pod:
+
+{{< codenew file="pods/lifecycle-events.yaml" >}}
+
+No arquivo de configuração, você pode ver que o comando `postStart` escreve um arquivo 
+`message` no diretório `/usr/share` do contêiner. 
+O comando `preStop` encerra o nginx graciosamente. 
+Isto é útil se o contêiner é terminado devido a uma falha. 
+
+Crie o Pod:
+
+```shell
+    kubectl apply -f https://k8s.io/examples/pods/lifecycle-events.yaml
+```
+
+Verifique que o contêiner no Pod está executando:
+
+```shell
+    kubectl get pod lifecycle-demo
+```
+
+Inicie um shell no contêiner que está executando no seu Pod:
+
+```shell
+    kubectl exec -it lifecycle-demo -- /bin/bash
+```
+
+No seu shell, verifique que o manipulador `postStart` criou o arquivo `message`:
+
+```shell
+    root@lifecycle-demo:/# cat /usr/share/message
+```
+
+A saída mostra o texto escrito pelo manipulador `postStart`: 
+
+```shell
+    Hello from the postStart handler
+```
+
+<!-- discussion -->
+
+## Discussão
+
+O Kubernetes envia o evento `postStart` emediatamente após o Contêiner ser criado. 
```

Comment:
Hi @DonatoHorn, please, I thougth that´s a typo here about the word "_emediatamente_", change to "**imediatamente**".

Thanks